### DISCUSSION TITLE
Estimated release date for v3?

### DISCUSSION THREAD

**Discussion by JoshuaWise** [porsager/postgres#191] [General]
I'm very much looking forward to v2 because I require TypeScript definitions for my project. Any estimation on when it will be officially released?

**Comment by throrin19** [comment]
Any news of possible release date ?

**Comment by stalniy** [comment]
As far as I know, @porsager wants to add proper dynamic sql support and only then make a release of v2

**Comment by vafanassieff** [comment]
Just writting a comment to say that your lib is awesome and made writting sql enjoyable.

Waiting for v2 too, keep up the good work !

**Comment by karlhorky** [comment]
Would also love to see a non-beta v2 land!

> @porsager wants to add proper dynamic sql support and only then make a release of v2

@porsager is this accurate? Would it help to have any other maintainers help with getting the last few things done for v2?

**Comment by porsager** [comment]
Thanks a lot @vafanassieff, that means a lot!!

@karlhorky I'm considering moving the dynamic query improvements for another major, and get v2 out with the features currently in beta, and the outstanding bugs fixed. I think that should make it possible to get v2 out within a couple of weeks.

**Comment by karlhorky** [comment]
Amazing, thanks for the update!

Oh and I must also extend my kudos here 🙌 💯 Incredible work on this library.

Postgres.js makes SQL + JS really great to teach students (and is also awesome for use in our internal projects!)

**Comment by samuel-bach** [comment]
@porsager is the beta still being developed? It's been quite some time since the last release.
btw: you created a great lib : )

**Reply by porsager** [reply]
Thanks a lot Samuel. 

It’s ready to go as v3, but it’s just missing the last parts of the docs. Check out https://github.com/porsager/postgres/pull/259

**Reply by samuel-bach** [reply]
Uh nice! Have not seen the v3 before. Really love some of the new features
